app/src/routes/tfa-setup.vue

```diff
@@ -20,8 +21,19 @@ onMounted(() => {
 	}
 });
 
-const { generateTFA, enableTFA, loading, password, tfaEnabled, tfaGenerated, secret, otp, error, canvasID } =
-	useTFASetup(false);
+const {
+	generateTFA,
+	enableTFA,
+	cancel2FASetup,
```

Comment:
Example of the TFA-2FA inconsistency

```suggestion
	generateTFA,
	enableTFA,
	cancelTFASetup,
```

Reply:
I'm not entirely sure either, as it existed five years ago. https://github.com/directus/directus/blob/23857611f2119eb79d5adf19fcd7f38e77addd30/app/src/lang/translations/en-US.yaml#L353-L354